<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>530</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <item>
            <property name="text">
             <string>int</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>double</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>string</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="swap">
           <property name="styleSheet">
            <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
           </property>
           <property name="text">
            <string>Swap</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLineEdit" name="swapLine"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Для Swap (введите элементы и можете поменять местами с текущим вектором)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="output_2"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Insert(cначала индекс, потом значение), в Erase (cначала индекс, потом количество)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Resize(сначала новый размер, потом число которое будет добавляться(по умолчанию 0)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Assign(вводится значения)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="pushLine"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="push">
         <property name="styleSheet">
          <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
         </property>
         <property name="text">
          <string>Push back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pop">
         <property name="styleSheet">
          <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
         </property>
         <property name="text">
          <string>Pop back</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="empty">
           <property name="styleSheet">
            <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
           </property>
           <property name="text">
            <string>Empty</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="size">
           <property name="styleSheet">
            <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
           </property>
           <property name="text">
            <string>Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="capacity">
           <property name="styleSheet">
            <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
           </property>
           <property name="text">
            <string>Capacity</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="erase">
           <property name="styleSheet">
            <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
           </property>
           <property name="text">
            <string>Erase</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clear">
           <property name="styleSheet">
            <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="insert">
           <property name="styleSheet">
            <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
           </property>
           <property name="text">
            <string>Insert</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="resize">
       <property name="styleSheet">
        <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
       </property>
       <property name="text">
        <string>Resize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="assign">
       <property name="styleSheet">
        <string notr="true">float: rigth;
margin-right: 3%;
margin-top: 5px;
border-radius: 5px;
-moz-border-radius: 5px;
-webkit-border-radius: 5px;
padding: 5px 9px;
font-size: 1.2em;
background-color: rgba(236, 101, 80, 1);
text-shadow: #4545545 0 0 2px;
border-bottom: 4px solid rgba(217, 91, 72, 1);
color: white;
font-family: 'Roboto Slab', serif;</string>
       </property>
       <property name="text">
        <string>Assign</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
